apps/angular/src/app/features/authentification/login/login.component.ts

```diff
@@ -0,0 +1,100 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, DestroyRef, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormBuilder, FormGroup, ReactiveFormsModule, Validators } from '@angular/forms';
+import { Login } from '@js-camp/core/models/login';
+import { MatCardModule } from '@angular/material/card';
+import { MatButtonModule } from '@angular/material/button';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatIconModule } from '@angular/material/icon';
+import { BehaviorSubject, catchError, of, take } from 'rxjs';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { UserService } from '@js-camp/angular/core/services/user.service';
+import { Router } from '@angular/router';
+import { URL_PATHS } from '@js-camp/core/utils/url-paths';
+import { FormErrorService } from '@js-camp/angular/core/services/form-error.service';
+import { ApiError } from '@js-camp/core/models/api-error';
+
+import { AuthComponent } from '../auth.component';
+
+/** Login page component. */
+@Component({
+	selector: 'camp-login',
+	standalone: true,
+	templateUrl: './login.component.html',
+	styleUrl: './login.component.css',
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	imports: [
+		CommonModule,
+		ReactiveFormsModule,
+		MatCardModule,
+		MatButtonModule,
+		MatFormFieldModule,
+		MatInputModule,
+		MatIconModule,
+		AuthComponent,
+	],
+})
+
+export class LoginComponent {
+	private readonly destroyRef = inject(DestroyRef);
+
+	private readonly formBuilder: FormBuilder = inject(FormBuilder);
```

Comment:
This type annotation is unnecessary. It's okay to name `fb`.
```suggestion
	private readonly fb = inject(FormBuilder);
```

Also, we're usually using `NonNullableFormBuilder` and explicitly setting `null` for some fields if they're nullable.